package.json

```diff
@@ -40,15 +40,24 @@
     "@mantine/next": "^5.10.4",
     "@mantine/notifications": "^5.10.4",
     "@mantine/spotlight": "^5.10.4",
+    "@nivo/bump": "^0.80.0",
+    "@nivo/core": "^0.80.0",
+    "@nivo/line": "^0.80.0",
     "@tabler/icons": "^1.106.0",
+    "chart.js": "^4.2.1",
     "dayjs": "^1.11.7",
     "firebase": "^9.17.1",
     "json-2-csv": "^3.19.0",
+    "inherits": "^2.0.4",
+    "is-glob": "^4.0.3",
+    "jsesc": "^3.0.2",
     "lodash": "^4.17.21",
     "mantine-datatable": "^1.8.6",
     "next": "^13.1.6",
     "octokit": "^2.0.14",
+    "next-offline": "^5.0.5",
```

Comment:
Do we need this lib? It's notoriously outdated lib which several security holes.

Reply:
No, you can remove it if you want.